mscore/exportxml.cpp

```diff
@@ -277,12 +277,13 @@ class ExportMusicXml {
       Ottava const* ottavas[MAX_NUMBER_LEVEL];
       Trill const* trills[MAX_NUMBER_LEVEL];
       int div;
-      double millimeters;
-      int tenths;
+      double scale; // a multiplier, used in the getTenths functions
```

Comment:
This module has been refactored in a major way. The numeric formulas for calculating "tenths" and other units are completely redesigned and greatly simplified.
Some of the functions have also been generally simplified, beyond the numeric formulas. For example, they were passing around the class member variables as arguments all the time, for no reason. Only the member variables were ever passed in. So I removed the arguments and use the member variables instead.